op_builder/builder.py

```diff
@@ -218,12 +218,7 @@ def builder(self) -> Union["CUDAExtension", "CppExtension"]:
         from torch.utils.cpp_extension import CppExtension, CUDAExtension
 
         if self.ext_type == "cpp":
-            return CppExtension(
```

Comment:
It seems this "if branch" is empty

Reply:
`op_builder` is removed, everything is collated into `extensions`.

Reply:
can we remove this file?

Reply:
done